### PR TITLE
Add PolyMC

### PR DESCRIPTION
**Pull request by Kaydax**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/p/polymc.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  on_mojave :or_older do
+    sha256 "a92feb9747b34e1ff17d5546e56a2919aa491c558fe20b8f211cd8cdb0897434"
+
+    url "https://github.com/PolyMC/PolyMC/releases/download/#{version}/PolyMC-macOS-Legacy-#{version}.tar.gz",
+        verified: "github.com/PolyMC/PolyMC/"
+  end
+  on_catalina :or_newer do
+    sha256 "7fb72c2eef2900c90bc999062f85ed5e6faa60856bc7fa6e80e14c4bd4d4a61a"
+
+    url "https://github.com/PolyMC/PolyMC/releases/download/#{version}/PolyMC-macOS-#{version}.tar.gz",
```
Comment (Kaydax) [inline Casks/p/polymc.rb:13]: The app is signed, but not notarized

**Comment by miccal** [comment]
Thank you for your submission @Kaydax, but we cannot accept it at this time because it does not pass our signature verification check that are a requirement for new submissions:

> macOS on ARM requires software to be signed. Please contact the upstream developer to let them know they should sign and notarize their software.

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if the developers sign their app in the future we would then be happy to consider adding this to `homebrew-cask`.

**Comment by Kaydax** [comment]
> Thank you for your submission @Kaydax, but we cannot accept it at this time because it does not pass our signature verification check that are a requirement for new submissions:
> 
> > macOS on ARM requires software to be signed. Please contact the upstream developer to let them know they should sign and notarize their software.
> 
> Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if the developers sign their app in the future we would then be happy to consider adding this to `homebrew-cask`.

Thank you for the copy paste reply. I am one of the developers and would like to re-add my application back onto the repo. Is there no other alternatives as the application is fully signed and works on arm devices, its as I said in one of the comments that its just not notarized as:
1. Nobody on the development team has access to an apple developer account
2. Nobody is willing to pay over $100 a year just to be hosted on homebrew.

I saw nowhere demanding it has to be notarized in the docs about what a package can be rejected for. Is there any other options like blocking arm devices if that's the case?

**Comment by bevanjkay** [comment]
@Kaydax Apps that are not signed and notarized cannot be run on Macs with Apple Silicon hardware.
We are no longer accepting casks that don't work on the newer hardware, as it has been a few years since Apple has released any hardware that has Intel hardware.

> Nobody is willing to pay over $100 a year just to be hosted on homebrew.

I would suggest that this has much less to do with being hosted on Homebrew, but more to do with Apple users being able to use your application.

**Pull request closed without merging**